<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GDD8ABC" failures="1" tests="3" name="Test" time="13.676" errors="0" timestamp="2025-08-21T21:03:50 IST">
  <testcase classname="testNGclass2.ListenersTestNG" name="Testlogo" time="0.095"/>
  <testcase classname="testNGclass2.ListenersTestNG" name="testURL" time="0.029">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at testNGclass2.ListenersTestNG.testURL(ListenersTestNG.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testURL -->
  <testcase classname="testNGclass2.ListenersTestNG" name="testTiltle" time="0.0">
    <skipped/>
  </testcase> <!-- testTiltle -->
</testsuite> <!-- Test -->
